dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/deduplication/PotentialDuplicatesTests.java

```diff
@@ -140,34 +140,6 @@ public void shouldUpdateStatus( String status, String newStatus, boolean shouldU
         response.validate().statusCode( 400 ).body( "status", equalTo( "ERROR" ) );
     }
 
-    @Test
-    public void shouldGetDuplicatesByTei()
```

Comment:
this test should be changed to use `teis` query param instead. I can do it in your PR if you want, or you can do it :)